frontend/src/app/globals.css

```
@@ -976,29 +976,898 @@ input[type="reset"]:disabled {
+.landing-page .landing-surface-concept {
+  display: inline-flex;
+  align-items: center;
+  gap: 8px;
+  font-size: 0.84rem;
+  font-weight: 500;
+  color: var(--text);
+}
+
+.landing-page .landing-surface-question {
+  margin: 2px 0 4px;
+  font-family: var(--font-spectral), 'Spectral', Georgia, serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Spectral" (font-family-name-quotes)

(font-family-name-quotes)